scheduler/file.c

```diff
@@ -349,7 +349,7 @@ cupsdRemoveFile(const char *filename)	/* I - File to remove */
   * Then get the file size...
   */
 
-  if (fstat(fd, &info))
+  if (fstatat(fd, filename, &info, AT_SYMLINK_NOFOLLOW))
```

Comment:
Similarly, this doesn't actually do what you think. Also, since we opened the file in exclusive mode we know it isn't a symlink...